codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -315,10 +316,52 @@ class FluentClientGenerator(
     }
 
     private fun RustWriter.renderFluentBuilder(operation: OperationShape) {
+        val outputType = symbolProvider.toSymbol(operation.outputShape(model))
+        val errorType = symbolProvider.symbolForOperationError(operation)
         val operationSymbol = symbolProvider.toSymbol(operation)
+
         val input = operation.inputShape(model)
         val baseDerives = symbolProvider.toSymbol(input).expectRustMetadata().derives
         // Filter out any derive that isn't Clone. Then add a Debug derive
+        // input name
+        val fnName = clientOperationFnName(operation, symbolProvider)
+        implBlock(symbolProvider.symbolForBuilder(input)) {
+            fun takeOutWhereClause(s: String): String {
+                val n = s.indexOf("where\n")
+                if (n == -1) {
+                    return s
+                } else {
+                    return s.removeRange(0, n)
+                }
+            }
```

Comment:
Looks like this function is unused?
```suggestion
```

Reply:
Sorry, I forgot to remove it. I initially tried to remove `where` clause from the generated string.